### PR TITLE
Tag LowRankModels v0.2.0

### PR DESCRIPTION
**Pull request by madeleineudell**

### CODE REVIEW

**Comment by tkelman** [comment]
can you push the git tag so it shows up at https://github.com/madeleineudell/LowRankModels.jl/releases ?

**Comment by tkelman** [comment]
If you enabled Travis CI, would your tests pass there? The first line of `test/runtests.jl` looks like a typo so I'd guess not with this commit.

File: LowRankModels/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.3
```
Comment (tkelman) [inline LowRankModels/versions/0.2.0/requires]: Actually this fails its tests on 0.3 due to using `=>` in `fit_dataframe.jl`, ref https://travis-ci.org/tkelman/LowRankModels.jl/jobs/153518916. I'd recommend dropping support for Julia 0.3 going forward, instead set 0.4 as your minimum version here.

Comment (tkelman) [inline LowRankModels/versions/0.2.0/requires]: looks like you did on master with commit 3dd931c9f396eaf5eba1937b6335e0a985beb628, but that would need to be reflected here too

**Comment by tkelman** [comment]
closing this, please make a new branch for unrelated PR's

**Comment by madeleineudell** [comment]
@tkelman sorry, I though the prev commits had already been merged when I pushed to my branch. 

I'm not a git master; not sure how to split what I have onto separate branches. Can you tell me the best way to fix this?

**Comment by tkelman** [comment]
Sure.
```sh
cd ~/.julia/v0.5/METADATA
git remote add madeleineudell https://madeleineudell@github.com/madeleineudell/METADATA.jl
git fetch --all
git checkout origin/metadata-v2
git checkout -b tagconvex
git cherry-pick ... # insert sha's here that were tagging convex
git push madeleineudell tagconvex
# open a PR with the convex-specific commits
git checkout origin/metadata-v2
git checkout -b taglowrankmodels
git cherry-pick ... # insert sha's here that were tagging lowrankmodels
git push madeleineudell taglowrankmodels
# open a PR with the lowrankmodels-specific commits
```

**Pull request closed without merging**